core/common/boot/src/test/java/org/eclipse/edc/boot/system/ExtensionLoaderTest.java

```diff
@@ -98,6 +98,32 @@ void loadMonitor_whenNoMonitorExtension() {
         assertTrue(monitor instanceof ConsoleMonitor);
     }
 
+    @Test
+    void loadMonitor_programArgsSetConsoleMonitorLogLevel() {
+
+        var monitor = ExtensionLoader.loadMonitor(new ArrayList<>(), ConsoleMonitor.LEVEL_PROG_ARG + "=INFO");
```

Comment:
nit: this could be a `ParameterizedTest` with an `@EnumSource`

Reply:
[3b1253a](https://github.com/eclipse-edc/Connector/pull/4476/commits/3b1253a2a5d21ca8c584f8abf296ed69005e8727)